src/applications/vaos/components/FacilityPhone.jsx

```diff
@@ -13,6 +13,22 @@ export default function FacilityPhone({
   if (!contact) {
     return null;
   }
+
+  let number = contact;
+  let numberExtension = extension;
+
+  // Extract number and extension from contact if extension not explicitly set and
+  // contact contains an 'x' character, usually in the format "123-456-7890 x1234"
+  if (contact.includes('x')) {
+    const [contactNumber, contactExtension] = contact
+      .split('x')
+      .map(item => item.trim());
+    number = contactNumber;
+    if (!numberExtension) {
```

Comment:
I'm not sure what should happen if an extension is passed in via `extension` parameter and an extension exists in `contact`. Currently I set it up to prefer the explicitly passed `extension` but I'm not sure if that is the best resolution. Thoughts?

Reply:
The priority order to display phone number is Clinic phone, Facility phone then main VA phone. There will be cases where the clinic number will not have an extension. A better solution is to check for the value of `heading`. It will be either “Clinic phone”, “Phone” or “Main phone”. Checkout the helper layout function [DetailsPageLayout](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/components/layout/DetailPageLayout.jsx#L132)

Reply:
I've updated the logic here based on our offline conversation.